### PR TITLE
MK-521 - changed default schemaform

### PR DESCRIPTION
**Pull request by meek0**

### CODE REVIEW

File: mica-core/src/main/resources/config/project-form/definition.json [hunk 1]
```diff
@@ -31,7 +31,9 @@
               {
                 "key": "endDate",
                 "dateOptions": {
-                  "dateFormat": "dd-MM-yyyy"
+                  "dateFormat": "dd-MM-yyyy",
+                  "minDate": "startDate",
+                  "minDateIsRef": true
```
Comment (ymarcon) [inline mica-core/src/main/resources/config/project-form/definition.json]: What does this option mean?

Comment (meek0) [inline mica-core/src/main/resources/config/project-form/definition.json]: minDateIsRef denotes whether the property minDate is a reference to a form attribute or not.
If it is set to false or not set at all, minDate must be a string representation of the date. When minDateIsRef is true, the add-on will use minDate as key to find the real value.

Comment (ymarcon) [inline mica-core/src/main/resources/config/project-form/definition.json]: Is this a schemaform built-in thing?

Comment (meek0) [inline mica-core/src/main/resources/config/project-form/definition.json]: minDate is from the datapicker add-on and minDateIsRef is a new addition (it's already in release 0.1.3)

**Pull request merged**